src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.js

```diff
@@ -0,0 +1,80 @@
+import _ from 'lodash';
+import uiModules from 'ui/modules';
+import vislibSeriesTemplate from 'plugins/kbn_vislib_vis_types/controls/point_series/series.html';
+const module = uiModules.get('kibana');
+
+module.directive('vislibSeries', function ($parse, $compile) {
+  return {
+    restrict: 'E',
+    template: vislibSeriesTemplate,
+    replace: true,
+    link: function ($scope) {
+      function makeSerie(label) {
+        const last = $scope.series[$scope.series.length - 1];
+        return {
+          show: true,
+          mode: last ? last.mode : 'normal',
+          type: last ? last.type : 'line',
+          drawLinesBetweenPoints: true,
+          showCircles: true,
+          interpolate: 'linear',
+          data: {
+            label: label
+          },
+          valueAxis: $scope.vis.params.valueAxes[0].id
+        };
+      }
+
+      $scope.series = $scope.vis.params.seriesParams;
+      $scope.$watch(() => {
+        return $scope.vis.aggs.map(agg => {
+          try {
+            return agg.makeLabel();
+          } catch (e) {
+            return '';
```

Comment:
this happens when you select an aggregation and your field is not yet selected (makeLabel tries to access the field name)